app/src/server.js

```diff
@@ -0,0 +1,23 @@
+const { createServer } = require('https')
+const { parse } = require('url')
+const next = require('next')
+const fs = require('fs')
+const dev = process.env.NODE_ENV !== 'production'
+const app = next({ dev })
+const handle = app.getRequestHandler()
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0
+const httpsOptions = {
+  key: fs.readFileSync('../server.key'),
+  cert: fs.readFileSync('../server.crt'),
+}
+
+app.prepare().then(() => {
+  createServer(httpsOptions, (req, res) => {
+    const parsedUrl = parse(req.url, true)
+    handle(req, res, parsedUrl)
+  }).listen(3000, err => {
+    if (err) throw err
+    console.log('> Ready on https://localhost:3000')
```

Comment:
Shouldn't CI have failed with this `console.log` are we no longer enforcing that? Or are you ignoring it for this case and I'm just not seeing it?

Reply:
i was able to add this to the eslint config for the app directory but not graphql. for some reason the no-console rule won't play nicely with typescript files